backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MaintenanceNumberOfVdssCommand.java

```diff
@@ -486,7 +486,6 @@ private boolean validateNoActiveImageTransfers(VDS vds) {
                             .map(ImageTransfer::getDiskId)
                             .sorted()
                             .collect(Collectors.toList())));
-            return failValidation(EngineMessage.VDS_CANNOT_MAINTENANCE_HOST_WITH_RUNNING_IMAGE_TRANSFERS, replacements);
         }
```

Comment:
any reason to keep this method then?

Reply:
It seems to me like there is no need for this function. If you agree with me I will delete it

Reply:
yes, it can be dropped